ai/worker/utils.go

```diff
@@ -0,0 +1,29 @@
+package worker
+
+import (
+	"context"
+	"github.com/Masterminds/semver/v3"
+	"github.com/livepeer/go-livepeer/clog"
+)
+
+// LowestVersion returns the lowest version of a given pipeline and model ID from a list of versions.
+func LowestVersion(versions []Version, pipeline string, modelId string) string {
```

Comment:
I think it would be beneficial to add a log line somewhere that the Orchestrator has multiple versions.

Thinking if lowest test is true print a log line listing the current and lower versions with the pipeline and model_id in the log line. WDYT? I think this would help Orchestrators trouble shooting why not getting any jobs since there is no other log line that they are excluded from discovery.

Reply:
Added here: 7d9488bbc6074194abeb36529fd2e329610d2eeb PTAL

Reply:
I think this will work,